### ISSUE TITLE
Key binding not working after new tab

### ISSUE THREAD

**Issue by amir20** [jorgebucaran/fisher#465]
This is hard to explain because I am not sure at what point it stopped working. I noticed it when I had installed `jethrokuan/fzf` that nothing was working. 

Here is an example with `oh-my-fish/plugin-bang-bang`.

1. fisher add oh-my-fish/plugin-bang-bang
2. ls !! <--  This gets changed to `fisher add oh-my-fish/plugin-bang-bang`

Open a new tab and it doesn't work. 

I am going to guess that the key binding file is sourced the first time but not after that.

**Comment by jorgebucaran** [comment]
@amir20 Yes, please wait until https://github.com/jethrokuan/fzf/pull/85 is merged. Nothing to do with fisher itself.

Also, please read #448 to understand how key bindings are supported in fisher now.

**Comment by jorgebucaran** [comment]
@amir20 Also good to know that fisher supports oh-my-fish packages that provide custom key bindings, but you will need to include the dependency I mentioned in jethrokuan/fzf#85 for them to work on a new shell session. 

One more thing. Whenever you uninstall oh-my-fish/plugin-bang-bang, you will need to launch a new session to clear the bindings as there is no way to undo them automatically without using a <code>_pkg_\_uninstall</code> event handler which they aren't.

**Comment by amir20** [comment]
Ah I saw  jethrokuan/fzf#85 but thought it wasn't related to this.  i"ll wait until that's merged.

**Comment by jorgebucaran** [comment]
@amir20 See [configuration snippets](https://github.com/jorgebucaran/fisher#configuration-snippets) on the documentation too.
